src/libs/ReportUtils.js

```diff
@@ -1585,6 +1585,22 @@ function buildOptimisticReportPreview(chatReport, iouReport) {
     };
 }
 
+function updateReportPreview(iouReport, reportPreviewAction) {
```

Comment:
Missing JSDocs

Reply:
Good catch! Added for both `buildOptimisticReportPreview` and `updateReportPreview`